Source/WebCore/css/parser/CSSPropertyParserHelpers.cpp

```diff
@@ -7228,15 +7235,13 @@ RefPtr<CSSValue> consumePathOperation(CSSParserTokenRange& range, const CSSParse
 
 RefPtr<CSSValue> consumePath(CSSParserTokenRange& range, const CSSParserContext&)
 {
-    if (range.peek().type() != FunctionToken)
-        return nullptr;
-    if (range.peek().functionId() != CSSValuePath)
-        return nullptr;
-    auto args = consumeFunction(range);
-    auto result = consumeBasicShapePath(args, { });
-    if (!result || !args.atEnd())
-        return nullptr;
-    return result;
+    if (range.peek().type() == FunctionToken && range.peek().functionId() == CSSValuePath) {
```

Comment:
We generally favour early return for error (like the old code)

Reply:
Alright, I've rewritten it to use early returns in the latest patch.

Reply:
This code has been removed.